test/cypress/e2e/txDetails.spec.js

```diff
@@ -42,18 +48,34 @@ describe('Tx details', () => {
    * Delegate transaction details are shown and correct when tx is confirmed
    * @expect transfer details are correct for confirmed state
    */
-  it('Delegate vote', () => {
+  it('Vote', () => {
     cy.autologin(accounts.delegate.passphrase, networks.devnet.node);
     cy.visit(`${urls.wallet}?id=${delegateVoteTxId}`);
+    // Header
     cy.get(ss.txHeader).contains('Delegate vote');
-    cy.get(ss.txSenderAddress).should('have.text', accounts.delegate.address);
+    // Sender
+    cy.get(ss.txSenderAddress).should('have.text', accounts.delegate.address)
+      .click();
+    cy.get(ss.leftBlockAccountExplorer).find(ss.accountAddress).should('have.text', accounts.delegate.address);
+    cy.go('back');
+    // Recipient
     cy.get(ss.txRecipientAddress).should('not.exist');
+    // Date
     cy.get(ss.txDate).contains(/20\d\d$/);
-    cy.get(ss.txAddedVotes).contains(accounts.delegate.username);
+    // Added votes
+    cy.get(ss.txAddedVotes).contains(accounts.delegate.username)
+      .click();
+    cy.get(ss.leftBlockAccountExplorer).find(ss.accountAddress).should('have.text', accounts.delegate.address);
+    cy.go('back');
+    // Removed votes TODO add unvotes when Commander 2.0 will be free of bugs
     cy.get(ss.txRemovedVotes).should('not.exist');
+    // Fee
     cy.get(ss.txFee).should('have.text', '1');
+    // Confirmations
     cy.get(ss.txConfirmations).contains(/^\d/);
+    // Transaction ID
     cy.get(ss.txId).contains(regex.transactionId);
+    // Reference
```

Comment:
I don't see the added value of this and similar comments (e.g. `// Confirmations` vs. `ss.txConfirmations`)

Reply:
True. should I remove them?